addons/snailmail/static/src/components/message/message.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <t t-inherit="mail.Message" t-inherit-mode="extension">
+    <t t-name="snailmail.SnailmailMessage" t-inherit="mail.Message" t-inherit-mode="primary">
```

Comment:
Isn't this change preventing multiple modules from overriding the same component at the same time?

Reply:
Hmm, you're right. Maybe once we have xpath processing on the client side we could dynamically extend the template of the currently registered component so that we can keep the benefits I mentioned earlier, but in the meantime I will revert to patching the base component directly both in JS and XML.